soql-analyzer/src/main/scala/com/socrata/soql/typechecker/Typechecker.scala

```diff
@@ -161,6 +158,19 @@ class Typechecker[Type](typeInfo: TypeInfo[Type], functionInfo: FunctionInfo[Typ
     }
   }
 
+  private def checkWindowFunction(fc: FunctionCall, candidateFn: MonomorphicFunction[_]): Unit = {
```

Comment:
Is this always going to be a `MonomorphicFunction`? Why not `Function` here?

Reply:
We are hitch-hiking on the resolved funtions which are always Monomorphic.  We don't need another way to figure out the Functions(s).

Reply:
Good enough for me.